packages/help-center/src/components/help-center-article.scss

```diff
@@ -455,10 +455,70 @@
 				}
 			}
 
-			// Hide the tablist for the WP Support 3 tab component
-			// Context: https://github.com/Automattic/wp-calypso/issues/87340#issuecomment-1937424710
-			.wpsupport3-tab__tablist {
+			// Originally hidden with context: https://github.com/Automattic/wp-calypso/issues/87340#issuecomment-1937424710
+			// New CSS changes: Show the tab component for the WP Support 3 tab component
+
+			.wp-block-wpsupport3-tabs {
+
+				.aligncenter {
+					margin-left: auto;
+					margin-right: auto;
+				}
+				
+				.wpsupport3-tab__tablist {
+					display: flex;
+					align-items: center;
+					gap: 24px;
+					white-space: nowrap;
+					overflow-x: auto;
+					border-bottom: 1px solid var( --wp--preset--color--border-light-gray, #eee );
+
+					.wpsupport3-tab__title {
+						padding: 8px 0;
+			
+						color: var( --blue-blue-50, #0675c4 );
+						/* stylelint-disable-next-line declaration-property-unit-allowed-list */
+						font-size: 16px;
```

Comment:
I believe we don't need the 
/* stylelint-disable-next-line declaration-property-unit-allowed-list */
And we can use $font-body;